pkg/names/names.go

```diff
@@ -31,6 +31,8 @@ const (
 	OperatorComponent            = "submariner-operator"
 	ServiceDiscoveryCrName       = "service-discovery"
 	SubmarinerCrName             = "submariner"
+	MetricsProxyComponent        = "submariner-metrics-proxy"
```

Comment:
Although its mostly harmless, do you reckon we need to backport this to 0.13?

Reply:
Strictly speaking we don’t, but I think it’s easier if we just backport commits as-is as far as possible.